cmd/yace/main.go

```diff
@@ -118,9 +119,33 @@ func NewYACEApp() *cli.App {
 		},
 		&cli.IntFlag{
 			Name:        "cloudwatch-concurrency",
-			Value:       exporter.DefaultCloudWatchAPIConcurrency,
+			Value:       exporter.DefaultCloudwatchConcurrency.SingleLimit,
 			Usage:       "Maximum number of concurrent requests to CloudWatch API.",
-			Destination: &cloudwatchConcurrency,
+			Destination: &cloudwatchConcurrency.SingleLimit,
+		},
+		&cli.BoolFlag{
+			Name:        "cloudwatch-concurrency.per-api-enabled",
```

Comment:
@thepalbi wdyt about emitting a warning if the old and new CLI flags are used together?

e.g. setting both `-cloudwatch-concurrency` and `cloudwatch-concurrency.per-api-enabled` should log a warning message.

Reply:
Should also be documented that  `-cloudwatch-concurrency.per-api-enabled=true` ignores whatever value you pass to `-cloudwatch-concurrency`.